spk/mutt/src/service-setup.sh

```diff
@@ -0,0 +1,15 @@
+
+service_preupgrad ()
+{
+    # Save configuration
+    rm -rf ${TMP_DIR}/${PACKAGE}
+    mkdir -p ${TMP_DIR}/${PACKAGE}
+    mv ${INSTALL_DIR}/etc/Muttrc.local ${TMP_DIR}/${PACKAGE}/Muttrc.local
+}
+
+service_postupgrade ()
+{
+    # Restore configuration
+    mv ${TMP_DIR}/${PACKAGE}/Muttrc.local ${INSTALL_DIR}/etc/Muttrc.local
+    rm -rf ${TMP_DIR}/${PACKAGE}
```

Comment:
This very same save/restore logic comes in generic installer code for `var/` location.
So I would move muttrc configuration files there and document it in a wiki FAQ page for this package.